packages/core/src/stateUtils.ts

```diff
@@ -18,7 +18,9 @@ export const isLeafNode = (
 export function getChildren<TC, TE extends EventObject>(
```

Comment:
The biggest change caused by this that I've noticed is that now history states won't have `.order` set on them. That should be fine though because that's only used for sorting actions and history states can't have actions anyway